settings/src/lib.rs

```diff
@@ -121,14 +125,17 @@ pub struct NetworkSettings {
     /// This in memory variable specifies if we are a gateway or not
     #[serde(skip_deserializing, default)]
     pub is_gateway: bool,
+    /// How long do we wait without contact from a peer before we delete the associated tunnel?
+    #[serde(default = "default_tunnel_timeout")]
+    pub tunnel_timeout_seconds: u64,
 }
 
 impl Default for NetworkSettings {
     fn default() -> Self {
         NetworkSettings {
             own_ip: "fd00::1".parse().unwrap(),
             bounty_ip: "fd00::3".parse().unwrap(),
-            discovery_ip: Ipv6Addr::new(0xff02, 0x0, 0x0, 0x0, 0x0, 0x0, 0x1, 0x8),
+            discovery_ip: default_discovery_ip(),
```

Comment:
kinda curious how did you edit this but default _discovery_ip already existed? did I just miss it once?